### ISSUE TITLE
Crackling/distorted sound while playing Alice: Madness Retuns via Whisky

### ISSUE THREAD

**Issue by SpokNok0** [Whisky-App/Whisky#1067]
### Description

Hello everyone, lately I have been trying to play Alice: Madness Returns using Whisky.

The intro videos sounds just fine with no crackling, no problems at all, but just after loading into gaming menu, the sound becomes distorted, sound becomes even more distorted when i'm actually starting to play the game itself.

I'm using a Macbook Pro with M3 Pro processor with 18GB of RAM and the version of Whisky is 2.3.2.

Already tried using dsound, and using Windows 7/8 based bottle, nothing changes the audio is still awful.

Forgot to mention that the game itself runs just fine, no massive frame drops, and overall perfomance is good.

So i'm looking for solutions if there is any! Thanks.

### Steps to reproduce

none

### Expected behaviour

No sound distortion(crackling)

### Logs

```bash
No logs
```

### What version of Whisky are you using?

2.3.2

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by CharlyPico** [comment]
Yup this is also happening to me with Version 2.3.2 (40) of Whisky. The difference is that I'm playing Age of Mythology from Steam and that instead of crackles, the sound sometimes gets stuck and then un-stucks, and then stucks again. But the game runs with no issue at all. I'm also using the same Macbook Pro.

**Comment by EmilianoTalamo** [comment]
Have you tried the already known fixes? https://github.com/Whisky-App/Whisky/issues/637

There are plenty of tickets with fixes for audio for several engines. You should try those.

**Comment by SpokNok0** [comment]
> Have you tried the already known fixes? https://github.com/Whisky-App/Whisky/issues/637
> 
> 
> 
> There are plenty of tickets with fixes for audio for several engines. You should try those.

Yes, i've tried fixing it, but the fix mentioned in ticket #637 didn't change anything for me. And i haven't found any more fixes after looking on reddit+github. So i don't really know where do i find another solutions for my issue. :(

**Comment by smartaguillon** [comment]
This is also happening to me!! Please help!!

**Comment by smartaguillon** [comment]
Hello any update on the fix for this? Thank you!!

**Comment by CharlyPico** [comment]
For old games like Age Of Mythology, use Porting Kit App. It still has some frame stuttering when moving through the map, but there are no sound issues.

**Comment by lolokittycat** [comment]
yes, I've also been having _the same problem_... have you found a solution yet??

**Comment by Mateus109** [comment]
Yep, me too, I'm hearing the same crackling issue on a lot of games.

**Comment by sockmeowt** [comment]
ive been having the same problem on multiple games too. hopefully a solution can be found soon
